Makefile

```diff
@@ -197,7 +197,7 @@ docclean:
 RAWVER=$(shell $(PYTHON) tools/getnodeversion.py)
 VERSION=v$(RAWVER)
 FULLVERSION=$(VERSION)
-RELEASE=$(shell $(PYTHON) tools/getnodeisrelease.py)
+RELEASE=($shell grep "#define NODE_VERSION_IS_RELEASE" src/node_version.h | sed 's/[^0-9]//g')
```

Comment:
This can be shortened to `sed -ne 's/#define NODE_VERSION_IS_RELEASE \([01]\)/\1/p' src/node_version.h`